web/README.md

```diff
@@ -1,3 +1,324 @@
+
+**Return Type** : `void`
+
+---
+
+**Method** : `stop()`
+
+**Purpose** : Stop current animation
+
+**Return Type** : `void`
+
+---
+
+**Method** : `seek(frame: number)`
+
+**Purpose** : Move to given frame
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| src | `number` | The frame number to move, shouldn't be less than 0 and greater than `totalFrame`
+
+**Return Type** : `void`
+
+> [!TIP]
+> You can easily check total frame of animation, use `player.totalFrame`
+
+---
+
+**Method** : `destory()`
+
+**Purpose** : Destroy animation and lottie-player element
+
+**Return Type** : `void`
+
+---
+
+**Method** : `setLooping(value: boolean)`
+
+**Purpose** : Enable animation loop
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| value | `boolean` | `true` enables looping, while `false` disables looping.
+
+**Return Type** : `void`
+
+---
+
+**Method** : `setDirection(value: number)`
+
+**Purpose** : Set animation direction
+
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| value | `number` | Direction values. `1` to play the animation forward, `-1` to play it backward.
+
+**Return Type** : `void`
+
+---
+
+**Method** : `setSpeed(value: number)`
+
+**Purpose** : Set speed of animation
+
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| value | `number` | Playback speed. The value must be any positive integer.
+
+**Return Type** : `void`
+
+---
+
+**Method** : `setBgColor(value: string)`
+
+**Purpose** : Set a backgroud color
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| value | `string` | Color values. Hex(#fff) or string('red') to set background color
+
+**Return Type** : `void`
+
+---
+
+**Method** : `save(target: ExportableType)`
+
+**Purpose** : Export current animation to other file type
+
+**Parameters**
+| Name | Type | Description
+| --- | --- | --- |
+| value | `ExportableType` | File type to export
+
+**Return Type** : `void`
+
+
+---
+
+**Method** : `getVersion()`
+
+**Purpose** : Return current ThorVG version
+
+**Return Type** : `LibraryVersion`
+
+
+
+
+
+# Build
+This Lottie Player depends on ThorVG API, WASM build should be done before npm build. [[ref]](https://github.com/thorvg/thorvg/wiki/ThorVG-Viewer-Development-Guide#generate-webassembly-code)
+
+```sh
+# Build WASM
+cd ./thorvg
+rm -rf ./build_wasm
+./wasm_build.sh {absolute path}/emsdk/
+```
+
+Build web component
+
+```sh
+# Build library
+npm run build
+```
+You will have these files on `./dist` folder
+- `lottie-player.js` : web component for browser
+- `lottie-player.js.map` : source map
+- `lottie-player.esm.js` : web component for NPM (ECMAScript)
+- `lottie-player.esm.js.map` : source map
+- `lottie-player.cjs.js` : web component for NPM (CommonJS)
+- `lottie-player.cjs.js.map` : source map
+- `lottie-player.d.ts` : type declaration
+
+# Examples
+
+Please note these uses on various environments.
+
+- [VanillaJS Usage](https://codepen.io/Jinny-You/pen/ExrdaxX)
```

Comment:
Choosing the name 'thorvg-lottieplayer' over a personal name will contribute to a more professional image, aligning with the style of others. Could you please change it?